plugins/by-name/blink-cmp/default.nix

```
@@ -18,30 +18,62 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
   '';
 
   settingsOptions = {
-    keymap =
-      defaultNullOpts.mkNullableWithRaw (with types; either (attrsOf anything) (enum [ false ]))
-        {
-          show = "<C-space>";
-          hide = "<C-e>";
-          accept = "<Tab>";
-          select_prev = [
-            "<Up>"
-            "<C-p>"
-          ];
-          select_next = [
-            "<Down>"
-            "<C-n>"
-          ];
-          show_documentation = "<C-space>";
-          hide_documentation = "<C-space>";
-          scroll_documentation_up = "<C-b>";
-          scroll_documentation_down = "<C-f>";
-          snippet_forward = "<Tab>";
-          snippet_backward = "<S-Tab>";
-        }
-        ''
-          Customize the keymaps for blink.
-        '';
+    keymap = defaultNullOpts.mkNullableWithRaw' {
+      type = with types; either (attrsOf anything) (enum [ false ]);
+      pluginDefault = "default";
+      example = {
+        "<C-space>" = [
+          "show"
+          "show_documentation"
+          "hide_documentation"
+        ];
+        "<C-e>" = [ "hide" ];
+        "<C-y>" = [ "select_and_accept" ];
+
+        "<Up>" = [
+          "select_prev"
+          "fallback"
+        ];
+        "<Down>" = [
+          "select_next"
+          "fallback"
+        ];
+        "<C-p>" = [
+          "select_prev"
+          "fallback"
+        ];
+        "<C-n>" = [
+          "select_next"
+          "fallback"
+        ];
+
+        "<C-b>" = [
+          "scroll_documentation_up"
+          "fallback"
+        ];
+        "<C-f>" = [
+          "scroll_documentation_down"
+          "fallback"
+        ];
+
+        "<Tab>" = [
+          "snippet_forward"
+          "fallback"
+        ];
+        "<S-Tab>" = [
+          "snippet_backward"
+          "fallback"
+        ];
+
+      };
+      description = ''
+        The keymap can be:
+        - A preset ('default' | 'super-tab' | 'enter')
```

```suggestion
        - A preset (`'default'` | `'super-tab'` | `'enter'`)
```

Using markdown's `` `backticks` ``, we can mark this as a code snippet